resources/js/components/open/forms/OpenFormField.vue

```diff
@@ -232,7 +235,7 @@ export default {
         placeholder: field.placeholder,
         help: field.help,
         helpPosition: (field.help_position) ? field.help_position : 'below_input',
-        uppercaseLabels: this.form.uppercase_labels,
+        uppercaseLabels: this.form.uppercase_labels  == 1 || this.form.uppercase_labels == true,
```

Comment:
Why doing this?

Reply:
It was having a value of 0 at a lot of places and giving console errors because of type mismatch. Added this to get rid of those errors, shall I change it back?